docs/references/expo/custom-auth-flows/expo-impersonation.mdx

```diff
@@ -0,0 +1,491 @@
+---
+description: Learn how to build custom Impersonation into your Expo application
+---
+
+# Impersonation in Expo
+
+<Callout type="warning">
+The following flows are built assuming your Clerk app is using the default instance settings. If you have changed your instance settings, you may need to make changes to the examples.
+</Callout>
+
+User Impersonation is a powerful feature to support your customers with. Currently, from the Clerk Dashboard, [you can impersonate your users](https://clerk.com/docs/custom-flows/user-impersonation#impersonate-a-user-from-the-clerk-dashboard) and see through their eyes inside of your web applications. In this example, we’ll learn how to build impersonation into your mobile application through [Clerk’s custom flows](https://clerk.com/docs/custom-flows/overview). 
+
+The example we’ll build will still need some work before it goes into your app. For example, we will build impersonation directly into the dashboard, whereas you should build your impersonation into an admin only dashboard, where all of your users are listed & can be interacted with. You will want to restrict this so **only employees of your company or those who should have access to other user's information** can use this implementation.
+
+## Before you start
+
+Before continuing, please go through the [Expo Quickstart](https://clerk.com/docs/quickstarts/expo) and utilize the repo as a starting point for this example.
+
+```bash
+git clone https://github.com/clerk/clerk-expo-quickstart
+```
+
+Since we will be using `expo-router`'s experimental API routes, you will need to follow the [Expo docs](https://docs.expo.dev/router/reference/api-routes/#deployment) to deploy your routes to a domain. This is so we have access to our `CLERK_SECRET_KEY` that we need to pass to our impersonation endpoint. 
```

Comment:
```suggestion
1. Because this example uses [`expo-router`](https://docs.expo.dev/router/installation/)'s experimental API routes, you must follow the [Expo docs](https://docs.expo.dev/router/reference/api-routes/#deployment) to deploy your routes to a domain. Doing so allows Clerk to access to your `CLERK_SECRET_KEY`, which you must pass to your impersonation endpoint. 
```

- Markdown auto-numbers lists if every number is the same, so it's easiest to just make each number 1. That way if you delete an item or swap two items' places you don't have to renumber the list manually
- Removed "we" and "our"
- Added link to expo-router docs

Why does deploying your routes to a domain allow Clerk to access the secret key? Might be worth explaining here. What do you think?